src/Illuminate/Support/Str.php

```diff
@@ -1824,6 +1824,29 @@ public static function fromBase64($string, $strict = false)
         return base64_decode($string, $strict);
     }
 
+    /**
+     * Convert the given string to url safe Base64 encoding.
+     *
+     * @param  string  $string
+     * @return string
+     */
+    public static function toUrlSafeBase64($string): string
+    {
+        return str_replace(['+', '/', '='], ['-', '_', ''], base64_encode($string));
+    }
+
+    /**
+     * Decode the given url safe Base64 encoded string.
+     *
+     * @param  string  $string
+     * @param  bool  $strict
+     * @return string|false
+     */
+    public static function fromUrlSafeBase64($string, $strict = false)
+    {
+        return base64_decode(str_replace(['-', '_'], ['+', '/'], $string), $strict);
```

Comment:
You could use the `Str` classes method here
```suggestion
        return str_replace(['+', '/', '='], ['-', '_', ''], self::toBase64($string));
    }

    /**
     * Decode the given url safe Base64 encoded string.
     *
     * @param  string  $string
     * @param  bool  $strict
     * @return string|false
     */
    public static function fromUrlSafeBase64($string, $strict = false)
    {
        return self::fromBase64(str_replace(['-', '_'], ['+', '/'], $string), $strict);
```